src/Headers/Simulation.h

```diff
@@ -261,7 +266,7 @@ class Simulation : public SimulationBase
   virtual void ProcessNbodyParameters(void);
   virtual void ProcessParameters(void)=0;
   virtual void RecordDiagnostics(void);
-  virtual void RegulariseParticleDistribution(const int) {};
+  virtual void RegulariseInitialConditions(const int) {};
```

Comment:
A grep through the code tells me that this is called only for SPH. We should add it to the meshless